modules/hal_nxp/usb/usb_host_mcux_drv_port.h

```diff
@@ -0,0 +1,303 @@
+/*
+ * Copyright 2024 NXP
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Hi @danieldegrasse @dleach02 This file's content is from the MCUX SDK USB host stack files, so I use MCUX SDK license BSD-3. But the auto-check of zephyr report errors. How do I handle it? Thanks.

Reply:
Could we keep the file in the MCUX HAL? Otherwise I believe we would need a specific exemption to include this file in the Zephyr tree

Reply:
Move it to hal_nxp.